Day 1 exercise - installer, environment variables

davyjones1210 opened the pull request:
Creating first PR for Subin to review and accept

subing85 reviewed (approved):
Could you please remove all unknown files, if you need this code as a reference put out side the project

subing85 commented on bin/installer.py:
```diff
@@ -0,0 +1,24 @@
+# This is to setup the studio environment
+
+# import optparse
+# print(help(optparse))
+
+# import OptionParser class 
+# from optparse module.
+from optparse import OptionParser
+
+# create a OptionParser
+# class object
+parser = OptionParser()
+
+# add options
+parser.add_option("-e", "--environment",
+				dest = "environment",
+				help = "to set environment to our studio", 
+				metavar = "FILE")
+
+(options, args) = parser.parse_args()
+
+print(options, args)
+
+
```
```suggestion
# This is to setup the studio environment

#!/bin/bash

import logging
import optparse

logging.basicConfig(level=logging.INFO)

def execute():

    # python3 '/home/batman/Desktop/test/code_01.py'
    # python3 '/home/batman/Desktop/test/code_01.py' -e
    # python3 '/home/batman/Desktop/test/code_01.py' --open "belnder4.3.1"

    parser = optparse.OptionParser(
       usage="usage: %prog\n\tIntaller to set up my studio pipeline",
       version="0.0.1",
    )

    parser.add_option(
        "-e", "--environment",
        action = "store_true", 
        dest = "environment",
        default = False,
        help = "Set up the pipeline command and primary environments"
    )
    
    parser.add_option(     
        "-o", "--open",
        dest="open",
        action="store",
        help="To lauch(Open) DCC (Digital Command Control).",     
    )
     
    options, args = parser.parse_args()
 
    print(options, args)
    print("\n")
    
    if options.environment:
        print("\tenvironment", options.environment)
        logging.info("\tStart to setp my studio envs")
    
    if options.open:
        print("\tCurrent software", options.open)


if __name__ == "__main__":    
   execute()

```

davyjones1210 commented:
Hi @subing85, I have completed the homework assignment of debugging options.open in cat.py and also added add_option to store artist, project & show info within cat.py. See latest commit and push for changes. We will review in the call at 8pm this evening. Thanks

davyjones1210 commented:
Hi @subing85 , I have committed and pushed my updates from todays' rework. 

- Cleaned-up the utils.py and removed unnecessary print statements.
- Updated cat.py such that the user info is stored as environment variables. 
- Setup an environment variable for project and setup project path env variable
- Example: cat --artist "Kunal Dekhane" --project "PROJECT_ABC"
- Have not done any coding for domain, will do tomorrow after a few follow-up questions in today's call. 

See you at 8pm
